nodejs/src/services/AuthService.ts

```diff
@@ -119,7 +119,7 @@ export interface AppTokenConfig extends BaseTokenConfig {
   role: string;
 }
 
-export interface ManagementToken {
+export interface Token {
```

Comment:
why this rename? won't be ManagementToken be a clearer name as there are two types of tokens?

Reply:
`ManagementToken` type was being used for auth token too, so renamed it to be generic. Have fixed it now!